TeamMandate/Mandate.md

```diff
@@ -0,0 +1,53 @@
+# IT Strategy 
+
+## Vision  
+
+Create the Canadian Experience by leveraging modern technologies where government interacts seemlessly with Canadians through existing digital infrastructure 
+
+## Mission 
+
+Be the initiator of change within IITB to lay the path for the evolution of ESDCs digital services through an open collaborative approach to further ESDCs mandate in the digital era by alinging IITB with TBS Digital Standards 
+
+## Mandate 
+
+- Provide guidance to senior management on how IT can be leveraged to allow ESDC to deliver on its mandate 
+- Conduct both inward facing and external reaserch in order to provide evidence based aforementioned guidance 
+- Permeate and enforce modern IT practices and processes 
+
+## Guiding Principles 
+
+- **Add Value**
+
+⋅⋅⋅All efforts must deliver measurable impacts 
+⋅⋅⋅Implement strategies -- do not be a document factory 
+
+- **Work collaboratively**
+
+⋅⋅⋅Collaborate with anyone and everyone who can _add value_. This includes, but is not limited to, all Canadians, all levels of government, all departments, all private sector organizations, and all governments. 
+
+- **Continuous learning**
+
+⋅⋅⋅Learning must be a core part of daily activities to ensure up to date, modern, and well researched direction and strategy is provided to guide our actions. 
+
+- **Lead by Example**
+
+⋅⋅⋅By using the approaches and technologies we champion, we will be able to better respond to, and anticipate, the challanges that will be faced. 
+
+## Deliverables
+
+Demonstrating value of cutting edge approaches/methods **through action** with the aim of improving GC standards
```

Comment:
Change to
- Demonstrations of the value of cutting edge IT approaches and methods.
E.g. lead by example for XYZ

Recommendations of GC standards. 
E.g. GCCode as version control standard